solo/solo.py

```diff
@@ -123,6 +123,22 @@ def main():
         action="store_true",
         help="Get version of solo-sc",
     )
+    
+    parser.add_argument(
+        "--lr_st",
+        dest="lr_st",
+        default=1e-5,
+        type=int,
+        help="Learning rate used for solo.train",
+    )
+    parser.add_argument(
+        "--lr_vae",
+        dest="lr_vae",
+        default=1e-5,
```

Comment:
The code all runs fine. On my bench mark datasets these defaults lead to worse performance. Can we change them to `1e-3`?

Reply:
I only changed the lr_vae parameter and obtained the following results. Interestingly the results on training were slightly (0.01) lower, for validation results were slightly better. Good news: lr_vae=1e-3 ran through without any problems.

# LR 1e-3
Training results
AUROC: 0.96, Accuracy: 0.97, Average precision: 0.97
Validation results
AUROC: 0.90, Accuracy: 0.91, Average precision: 0.91

# LR 1e-5
AUROC: 0.96, Accuracy: 0.96, Average precision: 0.96
Validation results
AUROC: 0.92, Accuracy: 0.93, Average precision: 0.94

If it performs better on your benchmark dataset, I agree with you, lets go with 1e-3